tests/test_artifacts/cpdBootstrapUStarTh4Season20100901_artifacts/cpdBootstrapUStarTh4Season20100901_artifacts.json

```diff
@@ -0,0 +1,25 @@
+
+{
+  "test_cases": [
+    {
+      "id": "Testcase_1",
+      "description": "Basic functionality test using US_ARC_sample reference data",
+      "input": {
+        "0": "t.csv",
```

Comment:
It seems that `t.csv` is not in the `Testcase_1` folder. Is it because it has name conflict with `T.csv`? Maybe Github does not distinguish between lower and upper cases.

Reply:
I think this should be avoided as some filesystems won't distinguish uppercase from lowercase